plugins/woocommerce/includes/widgets/class-wc-widget-cart.php

```diff
@@ -43,6 +43,44 @@ public function __construct() {
 		parent::__construct();
 	}
 
+	/**
+	 * Get list of Woo blocks that update the cart.
+	 *
+	 * @return array;
+	 */
+	public static function get_blocks() {
+		return array(
+			'woocommerce/handpicked-products',
+			'woocommerce/single-product',
+			'woocommerce/product-collection',
+			'woocommerce/add-to-cart-form',
+			'woocommerce/product-top-rated',
+			'woocommerce/product-tag',
+			'woocommerce/products-by-attribute',
+			'woocommerce/product-on-sale',
+			'woocommerce/product-new',
+			'woocommerce/product-category',
+			'woocommerce/product-best-sellers',
+			'woocommerce/all-products',
+		);
+	}
+
+	/**
+	 * array_contains
+	 * Check if string contains word from array.
+	 *
+	 * @since 1.0.0
+	 * @version 1.0.0
```

Comment:
Linting for function fails because it has no PHPDocs for the params.

Reply:
Nice catch! I fixed it in the new function! 🙌